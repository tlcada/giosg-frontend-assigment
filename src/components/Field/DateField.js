import React from 'react';
import { TextField } from "@material-ui/core";

const DateField = (props) => {
    return(
        <TextField {...props} type="date" required margin="normal"
                   InputLabelProps={{
                       shrink: true
                   }}
        />
    )
};

export default DateField;
