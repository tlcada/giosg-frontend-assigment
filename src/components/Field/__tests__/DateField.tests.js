import React from 'react';
import { shallow } from 'enzyme';
import DateField from '../DateField';

describe('field', () => {
    it('renders without crashing', () => {
        const wrapper = shallow(<DateField/>);
        expect(wrapper).toMatchSnapshot();
    });
});
