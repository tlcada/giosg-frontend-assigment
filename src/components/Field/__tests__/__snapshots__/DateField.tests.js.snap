// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`field renders without crashing 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <DateField />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "InputLabelProps": Object {
        "shrink": true,
      },
      "margin": "normal",
      "required": true,
      "select": false,
      "type": "date",
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "InputLabelProps": Object {
          "shrink": true,
        },
        "margin": "normal",
        "required": true,
        "select": false,
        "type": "date",
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
